Components/navigation.css

```diff
@@ -0,0 +1,41 @@
+.navigation {
+    display: flex;
+    flex-direction: row;
+    background-color: var(--secondary-hover);
+    height: 2.5rem;
+    padding: 0.5rem 0.5rem;
```

Comment:
You can just mention `padding: 0.5rem` since it is the same for all sides

Reply:
thanks. i will do it.